src/routes/__tests__/routes.test.js

```diff
@@ -0,0 +1,12 @@
+import indexRoutes from '..';
+
+
+describe('Routes tests', () => {
+  indexRoutes.map((value, index) => {
+    it('should be an object of routes', () => {
+      expect(indexRoutes[index]).toHaveProperty('path');
+      expect(indexRoutes[index]).toHaveProperty('component');
+    });
+    return true;
```

Comment:
why do you have a return value in tests?

Reply:
Linting is suggesting that there should be a return. 
<img width="730" alt="screen shot 2018-11-28 at 11 16 51" src="https://user-images.githubusercontent.com/17355400/49138027-29ff9c80-f2ff-11e8-8a12-9adc90fd074d.png">
Am using map function to loop through all the route objects.

Reply:
you do not have to map over the index routes array, you can easily do 
`indexRoutes[0]` to access the object inside and assert for its properties.